### PR TITLE
Refactored the pingpong demo

### PR DESCRIPTION
**Pull request by kovacsgabor**
The pingpong example was very messy (as stated in #219) and therefore needed a major refactoring.

It was not the proposed solution in #219 but it makes that issue invalid.

### CODE REVIEW

File: examples/demo/pingpong/src/pingpong/j/model/associations/LeftPlayer.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package pingpong.j.model.associations;
+
+import hu.elte.txtuml.api.model.Composition;
+import pingpong.j.model.Player;
+import pingpong.j.model.Game;
+
+public class LeftPlayer extends Composition {
+	public class inGame extends HiddenContainer<Game> {
```
Comment (gerazo) [inline examples/demo/pingpong/src/pingpong/j/model/associations/LeftPlayer.java]: Why isn't it capital? Same true for player and PlayerOwnsRacket.player and .racket and for RightPlayer.

**Comment by gerazo** [comment]
Otherwise the above comment I like it, the names are much more descriptive than before.

**Comment by kovacsgabor** [comment]
We used to give non-capital names for association ends and connectors. This is against Java conventions but it makes the code more easily readable in my opinion.

Because this way instead of
```java
Action.link(LeftPlayer.InGame.class, game, LeftPlayer.Player.class, player);
```
we write
```java
Action.link(LeftPlayer.inGame.class, game, LeftPlayer.player.class, player);
```
which makes it clearer that this is the name of an association and its end.

**Comment by gerazo** [comment]
Ok, fair enough, let's merge!

**Pull request merged**